### ISSUE TITLE
Can't specify the scopes for new GCE instances.

### ISSUE THREAD

**Issue by ivannaranjo** [GoogleCloudPlatform/google-cloud-powershell#492]
Neither the `New-GceInstanceConfig` or `Add-GceInstance` have a `-Scope` parameter to specify the list of scopes to use for that instance.

There should be a param to specify what OAUTH scopes to authorize the instance for.

**Comment by quoctruong** [comment]
@ivannaranjo, you can supply a service account when creating instances. Service account can be built with [New-GceServiceAccountConfig](http://googlecloudplatform.github.io/google-cloud-powershell/#/google-compute-engine/GceServiceAccountConfig/New-GceServiceAccountConfig) and given the appropriate scopes. Is this what you are looking for?

**Comment by ivannaranjo** [comment]
No, that is not what I am looking for, as creating a service account just to specify the list of scopes is too much overhead. This will also require creating service accounts for every combination of scopes required.

Most of the time I just want to specify the scopes that I need, and let the tool use the default service account for me.

**Comment by quoctruong** [comment]
@ivannaranjo, I see. I can add a -Scope parameter to the cmdlets as another alternative for -ServiceAccount parameter.
